packages/playwright-utils/src/playwright-utils/fuelWalletTestHelper.ts

```diff
@@ -114,6 +114,13 @@ export class FuelWalletTestHelper {
     await approveButton.click();
   }
 
+  async walletSign() {
+    const walletPage = await this.getWalletPopupPage();
+
+    const approveButton = getButtonByText(walletPage, 'Sign');
```

Comment:
I think it's a good idea to check for "Sign" or other sign specific messages on the popup to make sure it's a sign message approval request